packages/main/src/plugin/proxy.ts

```
@@ -205,12 +205,24 @@ export class Proxy {
     // eslint-disable-next-line @typescript-eslint/no-this-alias
     const _me = this;
     globalThis.fetch = function (url: URL | RequestInfo, opts?: object): Promise<Response> {
-      const proxyurl = getProxyUrl(_me, asURL(url).protocol === 'https');
+      const urlObj = asURL(url);
+      const isHttps = urlObj.protocol === 'https:';
+      const proxyurl = getProxyUrl(_me, isHttps);
       if (proxyurl) {
         opts = {
```

Biome (2.1.2)

[error] Assigning a function parameter is confusing.

The parameter is declared here:

Developers usually expect function parameters to be readonly. To align with this expectation, use a local variable instead.

(lint/style/noParameterAssign)